core/commands/routing.go

```diff
@@ -21,9 +21,7 @@ import (
 	routing "github.com/libp2p/go-libp2p/core/routing"
 )
 
-var (
-	errAllowOffline = errors.New("can't put while offline: pass `--allow-offline` to override")
-)
+var errAllowOffline = errors.New("can't put while offline: pass `--allow-offline` to override")
```

Comment:
Altho I think this is up to judgment to use braces blocks for declaration, I think this is good, if you have only one thing don't put it in a brace block.